### PR TITLE
TESB-30238 Update of vulnerable Camel dependencies.

### PR DESCRIPTION
**Pull request by andreasmattes**
Update Camel dependencies with CVE.

### CODE REVIEW

**Review by jgreffe — APPROVED** [review]
lgtm 👌 
Please don't forget usage in RT repos

File: pom.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
   <properties>
     <upstream.version>2.25.2</upstream.version>
-    <apache-camel.features.tesb.version>2.25.2.tesb1</apache-camel.features.tesb.version>
+    <apache-camel.features.tesb.version>2.25.2.tesb2</apache-camel.features.tesb.version>
```
Comment (jgreffe) [inline pom.xml:44]: no need of upgrade here as 7.4.1 has not been released yet

**Review by jgreffe — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (andreasmattes) [reply pom.xml]: patch number increment reverted.

**Review by jgreffe — APPROVED** [review]
lgtm 👌

**Pull request merged**